libs/utils/WordPressUtils/src/main/java/org/wordpress/android/util/FormatUtils.java

```diff
@@ -32,4 +32,29 @@ public static final String formatInt(int value) {
     public static final String formatDecimal(int value) {
         return DECIMAL_INSTANCE.get().format(value).toString();
     }
+
+    /*
+     * returns the passed long formatted as an human readable filesize. Ex: 10 GB
+     * unitStrings is expected to be an array of all possible sizes from byte to TeraByte, in the current locale
+     */
+    public static final String formatFileSize(long size, final String[] unitStrings) {
+        final double log1024 = Math.log10(1024);
+        if (size <= 0) {
+            return "0";
+        }
+        int digitGroups = (int) (Math.log10(size) / log1024);
+
+        return String.format(unitStrings[digitGroups],
+                new DecimalFormat("#,##0.#").format(size / Math.pow(1024, digitGroups)));
```

Comment:
Previous comments got lost with new @mzorz additions to this PR. 
Should we replace `new DecimalFormat` with `NumberFormat.getInstance` here? 
See https://developer.android.com/reference/java/text/DecimalFormat.html for details.

Reply:
so, would this be the suggested change?

```
        NumberFormat f = NumberFormat.getInstance();
        if (f instanceof DecimalFormat) {
            ((DecimalFormat) f).applyPattern("#,##0.#");
        }
        return String.format(unitStrings[digitGroups], f.format(size / Math.pow(1024, digitGroups)));
```

Reply:
Don't have much experience with [NumberFormat](https://developer.android.com/reference/java/text/NumberFormat.html), but we can try by using methods like `setMinimumFractionDigits`, and `setMinimumIntegerDigits` if required.

Or doing something like you mentioned above. The docs says
` This will work for the vast majority of locales; just remember to put it in a try block in case you encounter an unusual one.
`
We don't need to catch exception since it's already tested before.

Reply:
Cool, I did this change in b5e1aa7 - I think that should be enough